app/src/main/java/cm/aptoide/pt/editorialList/EditorialListAdapter.java

```diff
@@ -84,7 +84,8 @@ public synchronized int getLoadingPosition() {
   }
 
   public void update(List<CurationCard> curationCards) {
-    this.editorialListItems = curationCards;
+    editorialListItems.clear();
+    editorialListItems.addAll(curationCards);
```

Comment:
I don't think this is the correct fix, it's simply a workaround. The real reason why there's two of the same editorial content is because on EditorialListRepository.loadNewEditorialListViewModel, there's no cloning being done after caching the editorialListViewModel. This means that the adapter is still referencing the cache list view model. The cloning is important to dereference it from the cache.

Additionally, while loadMore should add items to the list, a pullToRefresh or a first load should not add to adapter, but rather replace it completely (i.e. call the adapter.update())

For reference this is how it's being done in HomeBundles, see: BundlesRepository and respective adapter on HomeFragment.